content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,181 @@
+API currently supports Kubernetes 1.24+.
+
+### Expressiveness
+The Gateway API provides standard, Kubernetes-backed support for a wide range of features, such as
+header-based matching, traffic splitting, weight-based routing, request mirroring and more. With
+Ingress, these features need custom provider-specific annotations.
+
+### Extensibility
+Gateway API is designed with extensibility as a core feature. Rather than enforcing a
+one-size-fits-all model, it offers the flexibility to link custom resources at multiple layers
+within the API's framework. This layered approach to customization ensures that users can tailor
+configurations to their specific needs without overwhelming the main structure. By doing so, the
+Gateway API facilitates more granular and context-sensitive adjustments, allowing for a fine-tuned
+balance between standardization and adaptability. This becomes particularly valuable in complex
+cloud-native environments where specific use cases require nuanced configurations.
+
+
+## Upgrading to Gateway
+
+Migrating from Ingress to the Gateway API may seem intimidating, but luckily Kubernetes just
+released a tool to simplify the process.
+[ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway) assists in the migration by
+converting your existing Ingress resources into Gateway API resources. Here is how you can get
+started with Gateway API and using ingress2gateway:
+
+1. [Install a Gateway
+   controller](https://gateway-api.sigs.k8s.io/guides/#installing-a-gateway-controller) OR [install
+   the Gateway API CRDs manually](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api) 
+
+2. install [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway) If you have a Go
+development environment locally, you can install `ingress2gateway` with:
+
+```
+go install github.com/kubernetes-sigs/ingress2gateway@v0.1.0
+```
+
+This installs `ingress2gateway` to `$(go env GOPATH)/bin/ingress2gateway`.
+
+Alternatively, follow the installation guide
+[here](https://github.com/kubernetes-sigs/ingress2gateway#installation).
+
+3. Once the tool is installed, you can use it to convert the ingress resources in your cluster to
+   Gateway API resources.
+
+```
+ingress2gateway print
+```
+
+This above command will:
+
+1. Read your Kube config file to extract the cluster credentials and the current active namespace.
+1. Search for ingresses and provider-specific resources in that namespace.
+1. Convert them to Gateway API resources (Currently only Gateways and HTTPRoutes). For other options
+refer run the tool with `-h` or refer to https://github.com/kubernetes-sigs/ingress2gateway#options
+
+4. Review the converted Gateway API resources, validate them, and then apply them to your cluster.
+
+5. Send test requests to your Gateway to check that it is working. You could get your gateway
+   address using `k get gateway <gateway-name> -n <namespace> -o
```

Comment:
There's no `k` command as part of Kubernetes. Please clarify this.

Reply:
thanks for catching this!